packages/components/src/components/button/button.stories.ts

```diff
@@ -69,7 +70,62 @@ export const Primary: StoryObj = {
     disabled: false,
     'soft-disabled': false,
     variant: BUTTON_VARIANTS.PRIMARY,
+    size: PILL_BUTTON_SIZES[32],
+    color: BUTTON_COLORS.DEFAULT,
+  },
+};
+
+export const PrefixIcon: StoryObj = {
+  args: {
+    children: html`<mdc-icon name="info-circle-bold" slot="prefix-icon"></mdc-icon>
+      <span>Left Icon</span>`,
+    active: false,
+    disabled: false,
+    'soft-disabled': false,
+    variant: BUTTON_VARIANTS.PRIMARY,
+    size: PILL_BUTTON_SIZES[32],
+    color: BUTTON_COLORS.DEFAULT,
+  },
+};
+
+export const PostfixIcon: StoryObj = {
+  args: {
+    children: html`<p>Right Icon</p>
+      <mdc-icon name="info-circle-bold" slot="postfix-icon"></mdc-icon>`,
+    active: false,
+    disabled: false,
+    'soft-disabled': false,
+    variant: BUTTON_VARIANTS.PRIMARY,
+    size: PILL_BUTTON_SIZES[32],
+    color: BUTTON_COLORS.DEFAULT,
+  },
+};
+
+export const Icon: StoryObj = {
```

Comment:
in terms of naming the Buttons i would recommend to stick to the same naming as we are using in the Documentation, like Pill Button, Icon Button etc.